### DISCUSSION TITLE
how to use sakai-ng in my project and customize?

### DISCUSSION THREAD

**Discussion by aaron-za-dev** [primefaces/community#471] [PrimeNG Templates] [answered]
I want to use the sakai template (mainly the layout) in my project but I would like to remove some options such as the customization side menu (scale, themes...) and in the other side menu implement my own routes without using the existing code but I can't find any relevant documentation.

**Comment by mehmetcetin01140** [comment, marked as answer]
Hi,

This video explains how to do it : https://www.youtube.com/watch?v=yl2f8KKY204.
